app/src/main/java/com/github/geohunt/app/model/database/api/Claim.kt

```diff
@@ -20,7 +20,7 @@ interface Claim {
     /**
      * Unique identifier of the user that claimed the corresponding challenge
      */
-    val user: String
+    val user: LazyRef<User>
```

Comment:
Converted this to a LazyRef to keep the api consistent as we use a LazyRef for user in Challenge too